github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -55,7 +55,7 @@ presubmits:
         - "/bin/sh"
         - "-c"
         - |
-          bazelisk test --define tag=test //external-plugins/rehearse/... --test_output=all --test_arg=-test.v
+          bazelisk test //external-plugins/rehearse/... --test_output=all --test_arg=-test.v
```

Comment:
Can you move the test commands to the `.bazelrc` file?

Something like

```
test --test_output=all --test_arg=-test.v
```

should do the trick.

Reply:
Thanks a lot, done